index.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Please remove any comment from your code, to look like professional

Reply:
- [ ] Once again, great work using the inline comments. For this comment to be professional, kindly consider making the purpose of suggesting this change clear and understandable. I would also advise you to give detailed reasons why you are suggesting a change. 😎 👍 Please take a look at the following examples below:
Example of vague comment:
> The README is not descriptive, why don't you make it better?

Example of clear comment:
> The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?

Example of unexplained comment:
> Why don't you use an article tag here instead of a div?

Example of properly explained comment:
> Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible